README.md

```diff
@@ -42,7 +42,7 @@ Installation steps:
     * run `rake db:migrate`
     * OR
     * in MySQL, `drop database XXX;` for each database in `config/database.yml` and then try `rake db:setup` again
-* `rake db:seed` to populate it with initial data
+* `rake db:seed` to populate it with initial dummy data
```

Comment:
oh sorry it showed up in the same email thread for me. will do.

On Mon, Feb 23, 2015 at 11:53 AM, Bryan Bonvallet <notifications@github.com>
wrote:

> In README.md
> <https://github.com/publiclab/plots2/pull/235#discussion_r25179811>:
>
> > @@ -42,7 +42,7 @@ Installation steps:
> >      * run `rake db:migrate`
> >      * OR
> >      * in MySQL, `drop database XXX;` for each database in `config/database.yml` and then try `rake db:setup` again
> > -* `rake db:seed` to populate it with initial data
> > +* `rake db:seed` to populate it with initial dummy data
>
> woops we should bring the AGPL discussion back to the group. This is just
> in the diffs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/publiclab/plots2/pull/235/files#r25179811>.
>